src/test/java/com/epam/cube/logic/DirectorTest.java

```diff
@@ -0,0 +1,81 @@
+package com.epam.cube.logic;
+
+import com.epam.cube.data.DataReader;
+import com.epam.cube.entity.Cube;
+import com.epam.cube.entity.Point;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class DirectorTest {
+
+    public static final String FIRST_VALID_LINE = "2.1 3.3 -4.8. 3.0";
+    public static final String SECOND_VALID_LINE = "3.3 -1.2 -9.4 2.2";
+    public static final String FIRST_INVALID_LINE = "2.1z 3.3 -4.8. 3.0";
+    public static final String SECOND_INVALID_LINE = "2.1 3.3 -4.8. -3.0";
+    public static final Cube FIRST_VALID_CUBE = new Cube(
+            new Point(2.1, 3.3, -4.8), 3);
+    public static final Cube Second_VALID_CUBE = new Cube(
+            new Point(3.3, -1.2, -9.4), 2.2);
+    public static final String FILE_PATH = "filePath";
+
+    @Test
+    public void testReadShouldReturnListWhenDataAreValid() throws DataException{
+        //given
+        List<String> linesRead = Arrays.asList(FIRST_VALID_LINE, SECOND_VALID_LINE);
+
+        CubeValidator validator = Mockito.mock(CubeValidator.class);
```

Comment:
move to the top of the class as a constant, so as to make it visible what is mocked and what is tested